cmd/fastqFormat/fastqFormat.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/fastq"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func fastqFormat(s Settings) {
+	if !s.PairedEnd {
+		log.Fatalf("fastqFormat is still under development. Currently, the only formatting options available are for paired end reads. Select 'pairedEnd' from options.")
+	} else if s.SingleCell {
+		scPairChan := make(chan fastq.SingleCellPair)
+		go fastq.ReadToChanSingleCellPair(s.R1InFile, s.R2InFile, s.BarcodeLength, s.UmiLength, scPairChan)
+		outR1 := fileio.EasyCreate(s.R1OutFile)
+		defer outR1.Close()
```

Comment:
Just a reminder that deferring file closure is ok with os.File since the most serious error is "this file has already been closed", but the fileio writer has some serious errors that need to be handled.